automl/beta/data_residency.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def data_residency(project_id):
+    """Set a regional data endpoint using data residency"""
+    # [START automl_client_library]
+    # [START automl_data_residency]
+    from google.cloud import automl_v1beta1 as automl
+
+    client_options = {'api_endpoint': 'automl.googleapis.com:443'}
```

Comment:
Need to update to an actual regional endpoint here and in the other samples.

Reply:
So at the very least, this is the valid `default` endpoint.

Reply:
Done